package.json

```diff
@@ -12,14 +12,12 @@
     "gulp": "latest",
     "gulp-autoprefixer": "latest",
     "gulp-changed": "latest",
-    "gulp-concat": "latest",
     "gulp-filesize": "0.0.6",
     "gulp-filter": "latest",
     "gulp-imagemin": "latest",
     "gulp-jshint": "latest",
     "gulp-minify-css": "latest",
     "gulp-notify": "latest",
-    "gulp-rename": "latest",
```

Comment:
Can you confirm that we don't reference this and that any other node packages don't?